execution_engine/src/core/engine_state/mod.rs

```diff
@@ -604,8 +617,96 @@ where
 
         let gas_limit = Gas::new(U512::from(std::u64::MAX));
 
+        let wasmless_transfer_gas_cost = Gas::new(U512::from(
+            protocol_data.system_config().wasmless_transfer_cost(),
+        ));
+
+        let wasmless_transfer_cost = match Motes::from_gas(
+            wasmless_transfer_gas_cost,
+            WASMLESS_TRANSFER_FIXED_GAS_PRICE,
+        ) {
```

Comment:
> let wasmless_transfer_cost = match Motes::from_gas
recommend renaming to `wasmless_transfer_motes`

Even though, for all intents & purposes they have the same underlying value, it seems needlessly confusing IMO to create a Motes instance w/ this name when the config'd value (_protocol_data.system_config().wasmless_transfer_cost()_), is also named wasmless_transfer_cost but is expressed as Gas.